silver/tests/api/test_invoice.py

```diff
@@ -496,8 +503,8 @@ def test_issue_invoice_with_default_dates(self):
             'state': 'issued'
         }
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertTrue(all(item in response.data.items()
-                        for item in mandatory_content.iteritems()))
+        self.assertTrue(all(item in list(response.data.items())
+                        for item in six.iteritems(mandatory_content)))
```

Comment:
I reverted back the occurrences of six.iteritems since the compatibility script I used was attempting to replace model queries. Django "should" have these types of compatibilities concerns covered.

Reply:
`dict.iteritems()` is gone from Python3, so we're either stuck with `dict.items()` which in Python2 might take up some extra memory or `six.iteritems()` which uses the same implementation of dict.items()` from Python3.

Using `dict.items()` should be fine most of the times, especially in tests and places where we iterate over relatively few objects and 'six.iteritems()` is slightly harder to read.

What I should've said in the initial comment is that we should try to be consistent if possible.